### PR TITLE
chore(notice-comment): fix various copy issues

### PR DESCRIPTION
**Pull request by byhow**
re: #4460

### CODE REVIEW

File: lang/zh-Hans.json
Comment (robertu7) [inline lang/zh-Hans.json:1]: pls correct the translations

File: lang/zh-Hant.json
Comment (robertu7) [inline lang/zh-Hant.json:1]: please correct the translations

File: src/components/Notice/NoticeActorName.tsx [hunk 1]
```diff
@@ -27,7 +27,9 @@ const NoticeActorName = ({
         className={styles.displayName}
         data-test-id={TEST_ID.NOTICE_USER_DISPLAY_NAME}
       >
-        {user.displayName}
+        {user.displayName && user.displayName.length > 20
```
Comment (robertu7) [inline src/components/Notice/NoticeActorName.tsx:30]: can use `MAX_USER_DISPLAY_NAME_LENGTH` from `utils/enums` instead

**Review by robertu7 — CHANGES_REQUESTED** [review]

**Comment by byhow** [comment]
@robertu7 Sorry the translation bit passed through, prolly cuz I missed it after it changed after the i18n check or something. Ready for review again!

**Review by robertu7 — APPROVED** [review]

**Pull request merged**